packages/ckeditor5-basic-styles/package.json

```diff
@@ -16,6 +16,7 @@
     "@ckeditor/ckeditor5-core": "44.1.0",
     "@ckeditor/ckeditor5-typing": "44.1.0",
     "@ckeditor/ckeditor5-ui": "44.1.0",
+    "@ckeditor/ckeditor5-utils": "44.1.0",
```

Comment:
This is used only in tests. I'm not sure if those should not be dev dependencies.

Reply:
It's also used in typings for translations in the dist folder, so it's a production dependency:

```ts
import type { Translations } from '@ckeditor/ckeditor5-utils';
declare const translations: Translations;
export default translations;
```